modules/ingester/instance_search.go

```diff
@@ -485,6 +496,12 @@ func (i *instance) SearchTagValuesV2(ctx context.Context, req *tempopb.SearchTag
 				return err
 			}
 			span.SetAttributes(attribute.Bool("cached", true))
+			// instead of the reporting the InspectedBytes of the cached response,
+			// report the size of cacheData as the Inspected bytes
+			// because it's incorrect the report the metrics of the cachedResponse so
+			// we report the size of the cacheData to give an idea of how much data was read.
+			mc.Add(uint64(len(cacheData)))
```

Comment:
Hmm this is interesting.  Good question about if the throughput is before or after cache...